src/EmotionRecognitionTask.ts

```diff
@@ -42,16 +40,23 @@ export default async function emotionRecognitionTask(onFinish?: (data: EmotionRe
     language: string;
   };
 
+  type EmotionRange = "correctResponse" | "welcome" | "loadingStimulus" | "guidelines" | "initialInstructions" | "audioInstructions" | "videoTaskInstructions" | "audioVisualTaskInstructions" | "buttonSelectionWarning" | "examplePrompt" | "emotions.Anger" | "emotions.Fear" | "emotions.Contempt" | "emotions.Interest" | "emotions.Joy" | "emotions.Pride" | "emotions.Pleasure" | "emotions.Relief" | "emotions.Sadness" | "emotions.Disgust"
+
+  type JsonResult = {
+    timestamp: string;
+    [key: string]: any; // or define other known properties instead of `any`
+  }
   // parse settings
   const settingsParseResult = $Settings.safeParse(experimentSettingsJson);
+ 
 
   if (!settingsParseResult.success) {
     throw new Error(
-      `validation error, check experiment settings \n error can be seen below: \n ${settingsParseResult.error}`
```

Comment:
shouldn't this be displayed to the user somewhere? I don't think Maxime will know to check the console.